src/themes/components/index.ts

```diff
@@ -19,6 +19,7 @@ import Divider from './divider';
 import Checkbox from './checkbox';
 import Carousel from './carousel';
 import Button from './button';
+import Breadcrumbs from './breadcrumbs';
```

Comment:
I assume you wanna export the Item and Link components as well so we can use them in the projects